src/implementations/vanilla/components/gallery/gallery.js

```diff
@@ -357,11 +357,27 @@ export class Gallery {
    */
   updateOverlay(selectedItem) {
     this.selectedItem = selectedItem;
+    const embeddedVideo = selectedItem.dataset.eclGalleryItemEmbedSrc;
```

Comment:
You can use `getAttribute()` in a similar way as the `setAttribute()`. The `dataset`:

```js
const embeddedVideo = selectedItem.getAttribute('data-ecl-gallery-item-embed-src')
```

It'll be also more consistent to the following used in the same file

```javascript
const shareHref = this.selectedItem.getAttribute('data-ecl-gallery-item-share');
```